web/packages/teleterm/src/ui/DocumentTerminal/Terminal/ctrl.ts

```diff
@@ -51,7 +51,8 @@ export default class TtyTerminal {
   open(): void {
     this.term = new Terminal({
       cursorBlink: false,
-      fontFamily: this.options.fontFamily,
+      fontFamily: this.el.style.fontFamily, // grab font family from style to prevent injecting malicious CSS
```

Comment:
Let's add a link to that frontarm.com blogpost here as well.

Reply:
Done